design_docs/logging_subsystem.md

```diff
@@ -8,7 +8,7 @@ It is designed to satisfy following requirements.
 - The existing scenario, especially an input ovirt + output elasticsearch should not be affected by the updates.
 
 ```
-logging/
+logging
```

Comment:
Is it useful to have this long directory tree listing in this file that you must update every time you change the role structure? Maybe consider removing it in some future PR? @jvymazal @richm do you find useful to have this in the document?

Reply:
For me, probably not that much, if I need it, I can always browse it either here or locally.

Reply:
maybe a description of the general layout, but not the specific files e.g. logging -> roles -> rsyslog -> tasks -> output -> task file for each output